app/views/projects/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1><%= @project.name %></h1>
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count %></p>
```

Comment:
Doing `.count` is a calculation method, similar to `.average`, and as such should be in a model method and tested in a model test. This is also an MVC infraction.